librz/include/rz_bin.h

```diff
@@ -295,7 +295,7 @@ struct rz_bin_file_t {
 	RzList *xtr_data;
 	Sdb *sdb;
 	Sdb *sdb_info;
-	Sdb *sdb_addrinfo;
+	RZ_DEPRECATE Sdb *sdb_addrinfo; //< deprecated to use for new code, should be refactored
```

Comment:
Just for the record, the issue was opened here: https://github.com/rizinorg/rizin/issues/843